Index: jq-1.4/libm.h
===================================================================
--- jq-1.4.orig/libm.h
+++ jq-1.4/libm.h
@@ -10,8 +10,10 @@ LIBM_DD(cosh)
 LIBM_DD(exp2)
 LIBM_DD(exp)
 LIBM_DD(floor)
+/*
 LIBM_DD(j0)
 LIBM_DD(j1)
+*/
 LIBM_DD(log10)
 LIBM_DD(log2)
 LIBM_DD(log)
@@ -21,8 +23,10 @@ LIBM_DD(sqrt)
 LIBM_DD(tan)
 LIBM_DD(tanh)
 LIBM_DD(tgamma)
+/*
 LIBM_DD(y0)
 LIBM_DD(y1)
+*/
 /* LIBM_DID(jn) */
 /* LIBM_DID(yn) */
 /* LIBM_DDD(pow) */
